### PR TITLE
add send invite to Customer service.

### PR DESCRIPTION
**Pull request by djm204**
Needed for sending an invite to the customer AFTER the order has been created.

### CODE REVIEW

File: src/Services/Customer.php [hunk 1]
```diff
@@ -132,4 +132,12 @@ public function delete(ShopifyCustomer $customer)
     {
         return $this->client->delete("admin/customers/{$customer->getId()}.json");
     }
+
+    /**
+     * @param int $shopifyCustomerId
+     * @return array
+     */
+    public function sendAccountCreationInvite(int $shopifyCustomerId) {
```
Comment (rickywiens) [inline src/Services/Customer.php]: Code formatting

File: src/Services/Customer.php [hunk 2]
```diff
@@ -132,4 +132,12 @@ public function delete(ShopifyCustomer $customer)
     {
         return $this->client->delete("admin/customers/{$customer->getId()}.json");
     }
+
+    /**
+     * @param int $shopifyCustomerId
+     * @return array
```
Comment (rickywiens) [inline src/Services/Customer.php]: Code formatting

File: src/Services/Customer.php (hunk 1 above)
Comment (jerairrest) [inline src/Services/Customer.php]: no php 7 typehinting plz

File: src/Services/Customer.php [hunk 3]
```diff
@@ -132,4 +132,12 @@ public function delete(ShopifyCustomer $customer)
...
+     */
+    public function sendAccountCreationInvite(int $shopifyCustomerId) {
+        return $this->client->post("admin/customers/{$shopifyCustomerId}/send_invite.json", [], []);
```
Comment (rickywiens) [inline src/Services/Customer.php]: This should probably implement the whole endpoint, including the customizable parameters.

File: src/Services/Customer.php (hunk 1 above)
Comment (djm204) [reply src/Services/Customer.php]: done

File: src/Services/Customer.php (hunk 2 above)
Comment (djm204) [reply src/Services/Customer.php]: fixed

File: src/Services/Customer.php (hunk 1 above)
Comment (djm204) [reply src/Services/Customer.php]: done

File: src/Services/Customer.php (hunk 3 above)
Comment (djm204) [reply src/Services/Customer.php]: added

File: src/Services/Customer.php [hunk 4]
```diff
@@ -132,4 +132,15 @@ public function delete(ShopifyCustomer $customer)
...
     }
+
+    /**
+     * @param int $shopifyCustomerId
+     * @param array $params
+     * @param array $body
+     *
+     * @return array
+     */
+    public function sendAccountCreationInvite($shopifyCustomerId, $params, $body) {
```
Comment (jerairrest) [inline src/Services/Customer.php]: Still should have default params if they're not needed.

Comment (djm204) [reply src/Services/Customer.php]: right... will add default

File: src/Services/Customer.php [hunk 5]
```diff
@@ -132,4 +132,15 @@ public function delete(ShopifyCustomer $customer)
...
     }
+
+    /**
+     * @param int $shopifyCustomerId
+     * @param array $params
+     * @param array $body
+     *
+     * @return array
+     */
+    public function sendAccountCreationInvite($shopifyCustomerId, $params = [], $body = []) {
```
Comment (rickywiens) [inline src/Services/Customer.php]: code formatting still

Comment (djm204) [reply src/Services/Customer.php]: Diggity Donezo

**Pull request merged**